libminifi/test/unit/LoggerTests.cpp

```diff
@@ -276,7 +280,7 @@ TEST_CASE("Setting either properties to 0 disables in-memory compressed logs", "
   log_config.initialize(properties);
   auto logger = log_config.getLogger("DisableCompressionTestLogger");
   logger->log_error("Hi there");
-  REQUIRE((logging::LoggerConfiguration::getCompressedLog(true) == nullptr) == is_nullptr);
+  REQUIRE(logging::LoggerConfiguration::getCompressedLogs(true).empty() == is_nullptr);
```

Comment:
it would be better to rename `is_nullptr` to `is_empty`

Reply:
Updated in d32b92c94c133b8baf0d81be52bb780da799f05e